### PR TITLE
solve rubocop offences occuring on bundle update

### PR DESCRIPTION
**Pull request by lokeshh**
After doing bundle update or bundle install on a new system (with no daru) one is getting rubocop failures on master branch. This PR solves those issues.

### CODE REVIEW

**Comment by lokeshh** [comment]
There are two rubocop offences still unresolved. That's why this PR is in WIP. Resolving them now.

**Comment by lokeshh** [comment]
Solved the last two offences but there's a [this](https://gist.github.com/lokeshh/699f98f36814b339cc9ec68ed3b62362) example failing.

**Comment by lokeshh** [comment]
It's a result of calling `MultiIndex#include?` with a non array object. So, I've added a check in the method to resolve this.

File: lib/daru/dataframe.rb [hunk 1]
```diff
@@ -1817,10 +1817,14 @@ def method_missing(name, *args, &block)
       elsif has_vector? name
         self[name]
       else
-        super(name, *args, &block)
+        super
```
Comment (v0dro) [inline lib/daru/dataframe.rb]: why dont you need to pass the arguments anymore?

Comment (zverok) [inline lib/daru/dataframe.rb]: @v0dro `super` without arguments is called implicitly with _exactly_ the arguments of current method

**Comment by v0dro** [comment]
Yes. #187 can be addressed in a separate PR.

**Pull request merged**